molecule/default/verify.yml

```diff
@@ -5,10 +5,58 @@
   hosts: all
```

Comment:
It won't work because pipe symbol is missing. Should be:
```
- name: send put request
  ansible.builtin.shell: |
    etcdctl put foo bar
  register: etcd_put_reply
```